helm-charts/iqserver/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+appVersion: 1.84.0
+description: Sonatype Nexus IQ Server continuously monitors your entire software supply
+  chain
+home: https://www.sonatype.com/nexus-iq-server
+icon: https://www.sonatype.com/hubfs/2019%20Horizontal%20Product%20logo/horizontal%20Repo%20/Repo/NexusRepo_horiz.svg
```

Comment:
just to confirm - you want to use the long, horizontal logo rather than the stacked one?

Reply:
this got copied in here directly from the helm chart by operator-sdk. I don't think it matters at all, as it shows the icon I had to manually set in the CSV file.